.clang-format

```diff
@@ -15,7 +15,6 @@ SpaceBeforeCtorInitializerColon: true
 SpaceBeforeInheritanceColon: true
 SpacesInAngles: false
 SpacesInCStyleCastParentheses: false
-SpacesInConditionalStatement: false
```

Comment:
Was this included by accident, or did you intend to change the formatting rules here? If so, could you expand on why?

Reply:
It caused an error on my local installation and the script would not proceed.

Reply:
Perhaps you are using an older version of clang-format? We are using version 11 (`clang-format version 11.0.1`).

Reply:
Re-added the line.